### PR TITLE
checklist added

### PR DESCRIPTION
**Pull request by Y-Havryliv**
Checklist created.

### CODE REVIEW

File: checklist.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    SuperHeroKnightWithGun
+```
+
+3. Use descriptive error messages:
+
+Good example:
+
+```python
+if error:
+    raise CustomError("We have an error on the line 16.")
```
Comment (DmytroSvirsa) [inline checklist.md:50]: may be we can cower it with test?

Comment (Abnormaltype) [reply checklist.md:50]: good idea

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: checklist.md (hunk 1 above)
Comment (Y-Havryliv) [reply checklist.md:50]: but if I'm not mistaken, students can write different error messages

Comment (Abnormaltype) [reply checklist.md:50]: check, the message exists or not

File: tests/test_main.py [hunk 2]
```diff
@@ -43,8 +45,9 @@ def test_cafe_visit_should_raise_not_vaccinated_error_when_person_does_not_have_
     visitor,
 ):
     cafe = Cafe("")
-    with pytest.raises(NotVaccinatedError):
+    with pytest.raises(NotVaccinatedError) as error:
         cafe.visit_cafe(visitor)
+    assert str(error.value)
```
Comment (Abnormaltype) [inline tests/test_main.py:50]: please, describe this error

File: tests/test_main.py [hunk 3]
```diff
@@ -74,8 +77,9 @@ def test_cafe_visit_should_raise_outdated_vaccine_error_when_vaccine_has_expired
     visitor,
 ):
     cafe = Cafe("")
-    with pytest.raises(OutdatedVaccineError):
+    with pytest.raises(OutdatedVaccineError) as error:
         cafe.visit_cafe(visitor)
+    assert str(error.value)
```
Comment (Abnormaltype) [inline tests/test_main.py:82]: the same

File: tests/test_main.py [hunk 4]
```diff
@@ -105,8 +109,9 @@ def test_cafe_visit_should_raise_not_wearing_mask_error_when_wearing_a_mask_is_f
     visitor,
 ):
     cafe = Cafe("")
-    with pytest.raises(NotWearingMaskError):
+    with pytest.raises(NotWearingMaskError) as error:
         cafe.visit_cafe(visitor)
+    assert str(error.value)
```
Comment (Abnormaltype) [inline tests/test_main.py:114]: the same

**Review by Abnormaltype — APPROVED** [review]

**Review by DmytroSvirsa — APPROVED** [review]

**Pull request merged**